ui/src/components/DashboardMetrics.js

```diff
@@ -0,0 +1,234 @@
+            type: node.name,
+            y: item[1],
+          }));
+        }
+        return acc.concat(temp);
+      }, []);
+      return reduced;
+    },
+    [nodes],
+  );
+
+  // Passing nodes table as a react-queries identifier so if a node is added/removed the data are refreshed
+  // Also it makes the data to auto-refresh based on the node refresh timeout that is already implemented
+  const cpuDataQuery = useQuery(
+    ['dashboardMetricsCPU', nodes],
+    useCallback(
+      () =>
+        Promise.all(
+          nodes.map((node) =>
+            queryNodeCPUMetrics(node.internalIP, metricsTimeSpan),
+          ),
+        ).then((result) => formatNodesPromRangeForChart(result)),
+      [nodes, metricsTimeSpan, formatNodesPromRangeForChart],
+    ),
+  );
+
+  const memoryDataQuery = useQuery(
+    ['dashboardMetricsMemory', nodes],
+    useCallback(
+      () =>
+        Promise.all(
+          nodes.map((node) =>
+            queryNodeMemoryMetrics(node.internalIP, metricsTimeSpan),
+          ),
+        ).then((result) => formatNodesPromRangeForChart(result)),
+      [nodes, metricsTimeSpan, formatNodesPromRangeForChart],
+    ),
+  );
+
+  const loadDataQuery = useQuery(
+    ['dashboardMetricsLoad', nodes],
+    useCallback(
+      () =>
+        Promise.all(
+          nodes.map((node) =>
+            queryNodeLoadMetrics(node.internalIP, metricsTimeSpan),
+          ),
+        ).then((result) => formatNodesPromRangeForChart(result)),
+      [nodes, metricsTimeSpan, formatNodesPromRangeForChart],
+    ),
+  );
+
+  return (
+    <div id="dashboard-metrics-container">
+      <GraphWrapper className="cpuusage">
+        <GraphTitle>
+          <div>CPU Usage (%)</div>
+          {cpuDataQuery.isLoading && <Loader />}
+        </GraphTitle>
+        {!cpuDataQuery.isLoading && cpuDataQuery.isSuccess ? (
+          <LineChart
+            id={'dashboard_cpu_id'}
+            data={cpuDataQuery.data}
+            xAxis={xAxis}
+            yAxis={yAxisUsage}
+            color={perNodeColor}
+            width={graphWidth}
+            height={graphHeight}
+            lineConfig={lineConfig}
+            tooltip={true}
+            tooltipConfig={perNodeTooltip}
+            tooltipTheme={'dark'}
+          />
+        ) : null}
+      </GraphWrapper>
+      <GraphWrapper className="cpuusage">
+        <GraphTitle>
+          <div>Memory</div>
+          {cpuDataQuery.isLoading && <Loader />}
+        </GraphTitle>
+
+        {!memoryDataQuery.isLoading && memoryDataQuery.isSuccess ? (
+          <LineChart
+            id={'dashboard_memory_id'}
+            data={memoryDataQuery.data}
+            xAxis={xAxis}
+            yAxis={yAxisUsage}
+            color={perNodeColor}
+            width={graphWidth}
+            height={graphHeight}
+            lineConfig={lineConfig}
+            tooltip={true}
+            tooltipConfig={perNodeTooltip}
+            tooltipTheme={'dark'}
```

Comment:
We added a custom tooltip for vega-tooltip. 
Once we release a new version of `core-ui`, you can benefit from it.
```tooltipTheme={'custom'}```